debian/changelog

```diff
@@ -1,6 +1,7 @@
 matrix-synapse-py3 (1.31.0+nmu1) UNRELEASED; urgency=medium
 
   * Skip tests when DEB_BUILD_OPTIONS contains "nocheck".
+  * Update Debian build scripts for debhelper-compat level 12.
```

Comment:
this needs moving to the right version, since the world has moved on in the last couple of months